src/main/resources/modules/total_joint_replacement/functional_status_assessments.json

```diff
@@ -0,0 +1,1372 @@
+{
+  "name": "Functional Status Assessments",
+  "remarks": [
+    "This is a submodule that includes the possible assessments a patient will go through prior to and following either a total knee replacement or a total hip replacement.\n"
+  ],
+  "states": {
+    "KOOS_Total_Assessment": {
+      "type": "MultiObservation",
+      "category": "survey",
+      "number_of_observations": 5,
+      "codes": [
+        {
+          "system": "LOINC",
```

Comment:
The list of codes here should only contain the code(s) for the group as a whole, not the individual components (since those codes are listed on the individual observations themselves). In this case please just list the one for the top-level KOOS, not the individual scores. Same thing for the PROMIS, VR12, etc.

Reply:
Got it. I didn't realize it would group them without putting those in, but I understand it now so I'll fix that.